components/global.test.js

```diff
@@ -0,0 +1,97 @@
+import { shallow } from "enzyme";
+import   {MyApp } from "../pages/_app"
+import AppHead from "../Utils/layouts/head";
+import { PageFooter} from "../components/footer"
+import BreadCrumb, {BreadAnchor, BreadLink} from "../Utils/layouts"
+import * as nextRouter from 'next/router';
+
+
+describe("AppJs", () => {
+    it("renders footer component", () =>{
+        const app = shallow(<MyApp />).childAt(0).dive();
+        expect(app.containsMatchingElement(
+                <PageFooter/>
+        )).toBe(true);    
+    })
+    it("renders header component", () =>{
+        const app = shallow(<MyApp />).childAt(0).dive();
+        expect(app.containsMatchingElement(
+                <AppHead/>
+        )).toBe(true);    
+    })
+
+
+})
+
+
+
+
+
+describe("Breadcrumb", () => {
+
+    jest.mock("react-i18next", () => ({
+        withTranslation: () => (Component) => {
+          Component.defaultProps = { ...Component.defaultProps, t: () => [] };
+          return Component;
+        },
+      }));
```

Comment:
I’m not sure if this is doing anything. I assume it is to deal with translations but we don’t appear to have any tests that require this.

Reply:
Removing it is resulting in some inconsistent behaviours. Sometimes, it works and sometimes it does not. I believe it should be there because the `BreadCrumb` component expects it as a prop.

Reply:
OK, I guess it doesn't hurt to be there then. 👍